modules/web/src/app/shared/components/application-list/edit-application-dialog/component.ts

```diff
@@ -171,8 +172,18 @@ export class EditApplicationDialogComponent implements OnInit, OnDestroy {
           [Validators.required, KUBERNETES_RESOURCE_NAME_PATTERN_VALIDATOR, this._duplicateNameValidator()]
         )
       );
+      this.form.addControl(
+        Controls.AppResourcesNamespace,
+        this._builder.control(
+          {
+            value: this.application.spec.namespace?.name,
+            disabled: this.isEnforcedApplication(),
+          },
+          [KUBERNETES_RESOURCE_NAME_PATTERN_VALIDATOR]
```

Comment:
Two questions:

1- Can we edit the namespaces and the app name?
2- If so, shouldn't we add Validators.required?

Reply:
No, we can only edit application `values` field.